### ISSUE TITLE
Formatting partition with number in volume name - large size partition

### ISSUE THREAD

**Issue by mja65** [henrikstengaard/hst-imager#16]
If formatting a partition with a number (e.g. "work2") HST-imager hangs. Only option is to reboot my machine. If I keep all options the same other than the name, it works so issue seems specific to this.
This is on the windows version 

EDIT

It looks like this might relate to the size of the partition rather than the naming. If the HDF file is large (say 100GB) it will hang whereas it appears to work when the HDF image is smaller. I am unable to ascertain the cut off point unfortunately

**Comment by mja65** [comment]
Adding the steps to recreate issue on Windows:

hst.imager.exe blank Image.hdf 128GB
hst.imager.exe rdb init Image.hdf
hst.imager.exe rdb filesystem add Image.hdf pfs3aio pfs3
hst.imager.exe rdb part add Image.hdf DH0 pfs3 1gb
hst.imager.exe rdb part add Image.hdf DH1 pfs3 50gb
hst.imager.exe rdb part add Image.hdf DH2 pfs3 50gb
hst.imager.exe rdb part add Image.hdf DH3 pfs3 15gb

hst.imager.exe rdb part format Image.hdf 1 Workbench
hst.imager.exe rdb part format Image.hdf 2 Work
hst.imager.exe rdb part format Image.hdf 3 Stuff

It will hang on the last step.

**Comment by henrikstengaard** [comment]
I have tried the following steps on my Windows 11 computer with the newly released v1.2.390 and did not experience any hanging:

hst.imager.exe blank Image.hdf 128GB
hst.imager.exe rdb init Image.hdf
hst.imager.exe rdb filesystem add Image.hdf pfs3aio pfs3
hst.imager.exe rdb part add Image.hdf DH0 pfs3 1gb
hst.imager.exe rdb part add Image.hdf DH1 pfs3 50gb
hst.imager.exe rdb part add Image.hdf DH2 pfs3 50gb
hst.imager.exe rdb part add Image.hdf DH3 pfs3 15gb

hst.imager.exe rdb part format Image.hdf 1 Workbench
hst.imager.exe rdb part format Image.hdf 2 Work
hst.imager.exe rdb part format Image.hdf 3 Stuff
hst.imager.exe rdb part format Image.hdf 4 Stuff2

A lot of fixes have been included with the latest release and I would expect one of these fixed the hanging issue. Can you retry your steps with the newly released v1.2.390.

**Comment by henrikstengaard** [comment]
I have tried the following steps on my Windows 11 computer with the newly released v1.2.390 and did not experience any hanging:
```
hst.imager.exe blank Image.hdf 128GB
hst.imager.exe rdb init Image.hdf
hst.imager.exe rdb filesystem add Image.hdf pfs3aio pfs3
hst.imager.exe rdb part add Image.hdf DH0 pfs3 1gb
hst.imager.exe rdb part add Image.hdf DH1 pfs3 50gb
hst.imager.exe rdb part add Image.hdf DH2 pfs3 50gb
hst.imager.exe rdb part add Image.hdf DH3 pfs3 15gb

hst.imager.exe rdb part format Image.hdf 1 Workbench
hst.imager.exe rdb part format Image.hdf 2 Work
hst.imager.exe rdb part format Image.hdf 3 Stuff
hst.imager.exe rdb part format Image.hdf 4 Stuff2
```
A lot of fixes has been included with the latest release and I would expect one of these fixed the hanging issue. Can you retry your steps with the newly released v1.2.390.
